app/src/pages/projects/Project.tsx

```diff
@@ -39,14 +42,23 @@ const Project = () => {
   //get individual project data
   useEffect(() => {
     refreshProjectData(parseInt(projectId));
-  }, [projectId]);
+  }, []);
```

Comment:
Why have you removed projectId, this could cause issues as when the projectId it will not always trigger the refresh

Reply:
Oops, that was a mistake!